test/functional/core/startup_spec.lua

```diff
@@ -576,6 +577,48 @@ describe('user config init', function()
     eq(funcs.fnamemodify(init_lua_path, ':p'), eval('$MYVIMRC'))
   end)
 
+
+  describe 'with existing .exrc in cwd'(function()
+    local cwd = funcs.getcwd()
+    local exrc_path = '.exrc'
+    local xstate = 'Xstate'
+
+    before_each(function()
+      mkdir_p(xstate .. pathsep .. (is_os('win') and 'nvim-data' or 'nvim'))
+      write_file(exrc_path, [[
+        let g:from_exrc = 1
+        ]])
+    end)
+
+    after_each(function()
+      os.remove(exrc_path)
+      rmdir(xstate)
+    end)
+
+    it('loads .exrc', function()
+      clear{ args_rm = {'-u'}, env={ XDG_CONFIG_HOME=xconfig, XDG_STATE_HOME=xstate } }
+      command('terminal nvim')
+      local screen = Screen.new(80, 8)
+      screen:attach()
+      screen:expect{grid=string.format([[
+        ^                                                                                |
+                                                                                        |
+                                                                                        |
+                                                                                        |
+                                                                                        |
+        %s.exrc is not trusted.                             |
```

Comment:
This depends on the path and can easily fail. If the path is too long it may also take multiple lines. You can use `any = ` and `helpers.pesc` in `screen:expect()` instead. There is an example in `test/functional/ex_cmds/swapfile_preserve_recover_spec.lua`.

Reply:
Sure, I'll replace that. There's another issue that CI shows that I'm not sure how to tackle:

https://github.com/neovim/neovim/actions/runs/3548796299/jobs/5960439467#step:13:6990

Reply:
Use `helpers.nvim_prog` and pass arguments to it instead. Maybe `termopen()` can also be used instead of `:terminal`.

Reply:
I replaced it to use `nvim_prog`. `:terminal` seems to be working fine. Do you still want me to replace it with `termopen()` ?